esp-hal/src/gpio/lp_io.rs

```diff
@@ -257,6 +257,24 @@ macro_rules! lp_gpio {
                     }
                 }
             )+
+
+            #[doc(hidden)]
+            #[macro_export]
+            macro_rules! handle_rtcio {
```

Comment:
Next up, how can I delete these... I'm hoping there is a solution that allows me to factor this as a `repeat_erased!` and a `filter_branches!` macro, but I haven't tried anything yet.